addons/zylann.scatter/tools/plugin.gd

```diff
@@ -167,6 +167,7 @@ func _physics_process(delta):
 					var too_close = false
 					if len(_placed_instances) != 0:
 						var last_placed_transform = _placed_instances[-1].global_transform
+						if last_placed_transform.origin.distance_to(pos) < _pattern_margin + _palette.find_node("MarginSlider").value:
```

Comment:
Don't find child nodes of a foreign scene in this way. The node must be cached in `onready` and the value must be encapsulated with an accessor.

Reply:
Done.